### PR TITLE
Image gallery second attempt

### PR DESCRIPTION
**Pull request by psteyer**
Added a gallery section on the website.

It used a node back end making a GET call to Cloudinary to pull the pictures.
It then renders them in its own component under the Gallery section
Also includes Tailwind styling for the load more, grid styling for layout, and CSS.

### CODE REVIEW

**Comment by psteyer** [comment]
@cjmartian Looks like the checks passed, but I noticed there are still over 2,000 line additions. That doesn't seem right. Going to take a look into it. Could be due to file structure changes it's not ignoring some things that it needs to?

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+REACT_APP_API_URL=http://localhost:3500
+API_KEY=552737677385964
+CLOUD_NAME=dtvmeyqfx
+API_SECRET=1g_7AsLvmmaO11R_gohu-ErVks0
```
Comment (cjmartian) [inline .env]: What is this?
Looks like some secrets that shouldn't necessarily live openly on github.

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -9,7 +9,8 @@ function Footer() {
             <a
               href="#!"
               type="button"
-              className="rounded-full border-2 border-white text-white leading-normal uppercase hover:bg-black hover:bg-opacity-5 focus:outline-none focus:ring-0 transition duration-150 ease-in-out w-9 h-9 m-1">
+              className="rounded-full border-2 border-white text-white leading-normal uppercase hover:bg-black hover:bg-opacity-5 focus:outline-none focus:ring-0 transition duration-150 ease-in-out w-9 h-9 m-1"
+            >
```
Comment (cjmartian) [inline src/components/Footer.jsx:13]: Is this generated?

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -18,7 +19,8 @@ function Footer() {
                 className="w-2 h-full mx-auto"
                 role="img"
                 xmlns="http://www.w3.org/2000/svg"
-                viewBox="0 0 320 512">
+                viewBox="0 0 320 512"
+              >
```
Comment (cjmartian) [inline src/components/Footer.jsx:23]: same as above

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -29,7 +31,8 @@ function Footer() {
             <a
               href="#!"
               type="button"
-              className="rounded-full border-2 border-white text-white leading-normal uppercase hover:bg-black hover:bg-opacity-5 focus:outline-none focus:ring-0 transition duration-150 ease-in-out w-9 h-9 m-1">
+              className="rounded-full border-2 border-white text-white leading-normal uppercase hover:bg-black hover:bg-opacity-5 focus:outline-none focus:ring-0 transition duration-150 ease-in-out w-9 h-9 m-1"
+            >
```
Comment (cjmartian) [inline src/components/Footer.jsx:35]: Same

File: src/components/Footer.jsx [hunk 5]
```diff
@@ -49,7 +53,8 @@ function Footer() {
...
```
Comment (cjmartian) [inline src/components/Footer.jsx:57]: Same and so on, not going to comment on all of these.

**Review by cjmartian — COMMENTED** [review]
Left some comments, overall looks fine to me other than the .env file. Might need to rotate your keys and double check that stuff.

The bulk of the changes here are generated changes from the `package-lock.json` which is fine. Might be time to setup [dependabot](https://github.com/dependabot) on this repo.

File: .env (hunk 1 above)
Comment (psteyer) [reply .env]: ohwoops! 

I thought I added that file to .gitignore, I'll look into this and redo the credentials

File: src/components/Footer.jsx (hunk 2 above)
Comment (ous8292) [reply src/components/Footer.jsx:13]: That's tailwind CSS

Comment (psteyer) [reply src/components/Footer.jsx:13]: Line 12 is part of Tailwind CSS.

I believe line 13 is the closing tag for it.

File: src/components/Footer.jsx (hunk 3 above)
Comment (psteyer) [reply src/components/Footer.jsx:23]: Same here, line 23 is the closing tag

File: src/components/Footer.jsx (hunk 4 above)
Comment (psteyer) [reply src/components/Footer.jsx:35]: like 35 is a closing tag

File: src/components/Footer.jsx (hunk 5 above)
Comment (psteyer) [reply src/components/Footer.jsx:57]: All of those tags are closing tags for the <a> anchor element

File: .env (hunk 1 above)
Comment (cjmartian) [reply .env]: I saw that! Looks like this file was committed before the gitignore was, no worries just need to recreate your keys and delete this file and you should be good

File: src/components/Footer.jsx (hunk 2 above)
Comment (cjmartian) [reply src/components/Footer.jsx:13]: Ah makes sense to me, we can resolve these then

**Comment by ous8292** [comment]
@cjmartian This is ready to be merged!

**Comment by psteyer** [comment]
@cjmartian Would it make sense to create an example of the dotenv in the repo?

**Comment by cjmartian** [comment]
> @cjmartian Would it make sense to create an example of the dotenv in the repo?

Depends on how it's created. If it's done automatically then we should just document how that works.

**Comment by psteyer** [comment]
> Depends on how it's created. If it's done automatically then we should just document how that works.

I created it manually. I don't believe it would be treated automatically. 

I'll go ahead and merge for now.

**Review by cjmartian — APPROVED** [review]

**Pull request merged**